### PR TITLE
Run styler::style_pkg()

### PR DESCRIPTION
**Pull request by seabbs**
This PR demonstrates the changes suggested using automated styling. Note this does not fix all the current linting issues.

### CODE REVIEW

File: data-raw/config.R [hunk 1]
```diff
@@ -1,7 +1,6 @@
-
 susceptible <- 1L
-infectious  <- 2L
-removed     <- 3L
+infectious <- 2L
+removed <- 3L
```
Comment (pearsonca) [inline data-raw/config.R:3]: i expect there's not a linter option for aligned declaration / assignment, but i have a more than just mild preference for that style.

**Review by pearsonca — APPROVED** [review]
the one linter preference note aside, these all seem fine. if that's easy to set as an option, let's do it, otherwise fine.

File: data-raw/config.R (hunk 1 above)
Comment (seabbs) [reply data-raw/config.R:3]: I'm not sure - probably involves a bit of a dig in the `lintr` docs.

Comment (seabbs) [reply data-raw/config.R:3]: Sadly there is not and would also need to edit the style guide used by `styler`. Both of those things are doable but not trivially so.

**Comment by seabbs** [comment]
As above it sadly is not easy so going to go with this as is unless you say otherwise.

**Pull request merged**